src/Aspire.Hosting.SqlServer/SqlServerBuilderExtensions.cs

```diff
@@ -71,5 +71,21 @@ public static IResourceBuilder<SqlServerServerResource> WithDataVolume(this IRes
     /// <param name="isReadOnly">A flag that indicates if this is a read-only mount.</param>
     /// <returns>The <see cref="IResourceBuilder{T}"/>.</returns>
     public static IResourceBuilder<SqlServerServerResource> WithDataBindMount(this IResourceBuilder<SqlServerServerResource> builder, string source, bool isReadOnly = false)
-        => builder.WithBindMount(source, "/var/opt/mssql", isReadOnly);
+    {
+        // c.f. https://learn.microsoft.com/sql/linux/sql-server-linux-docker-container-configure?view=sql-server-ver15&pivots=cs1-bash#mount-a-host-directory-as-data-volume
```

Comment:
Why is this done only for `WithDataBindMount` and not for `WithDataVolume`?

Reply:
Binding specific folders only applies to folder mounts, not volumes. Volumes just work fine locally without changes, and the documentation only uses it for folders mounts too.

Reply:
@rwestMSFT - I see you wrote the documentation being referenced here in https://github.com/MicrosoftDocs/sql-docs/commit/51dddc6d9c9fd8fdeb9f78fec3172089c28bd5d7. Is there a reason we need to split all 3 directories when binding to a host directory, but when using a docker volume you just need a single volume? Why doesn't it work to map the single top-level `/var/opt/mssql` directory to the host machine?

cc @JerryNixon

Reply:
@eerhardt My name appears on thousands of articles. Please refer to the `author` and metadata when reviewing SQL Docs content.

Reply:
@amitkh-msft - Looks like you are the "author" of the article referenced above. Any chance you can answer the above question?

Reply:
Ack. Sure, please give me some time. I will look at this and try to answer the question.

Reply:
Any thoughts @amitkh-msft? Why can't we just bind mount to the root directory instead of the 3 separate ones?